### PR TITLE
Using concept list in concepticon

### PR DESCRIPTION
**Pull request by tresoldi**
As pointed on issue #3 

This is actually lowering the number of mapped concepts for some reason (would need to investigate manually, as the parameter ids changed and diff doesn't tell anything), but it is still better than having a local list (especially one out of sync with a list in Concepticon).

### CODE REVIEW

File: lexibank_yanglalo.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ def cmd_install(self, **kw):
 
                         for row in ds.add_lexemes(
                             Language_ID=language,
-                            Parameter_ID=parameter,
+                            Parameter_ID=concept_map[parameter],
                             Form=form,
                             Value=value,
                             Segments=segments,
```
Comment (LinguList) [inline lexibank_yanglalo.py]: Segments should be derived from the orthography profile, right? Could you please also check that this works and remove the explicit `self.tokenizer` line, as well as the explicit `form = `? This is one of the crucial check points as well. Let me know if this is unclear.

**Review by LinguList — CHANGES_REQUESTED** [review]
The concepticon list has more empty spots, because I removed many mappings in my latest PR to concepticon. I specifically did this to avoid too many mergers where the Chinese gloss would indicate different concepts.

File: lexibank_yanglalo.py (hunk 1 above)
Comment (tresoldi) [reply lexibank_yanglalo.py]: Sure, will do it.

**Comment by SimonGreenhill** [comment]
Where are we on this? looks like it's good to merge once rebuilt?

**Comment by tresoldi** [comment]
I suppose. I am rebuilding it.

**Comment by tresoldi** [comment]
Ok, I'm closing this PR without merging. @chrzyki had already pushed the code for using the concept-list back in November, this PR was out-of-sync. But I also recompiled the CLDF files and fixed the two things that were needed to get a green badge in the buildbot, it's already in `master`.

**Comment by SimonGreenhill** [comment]
Cool, thanks @tresoldi !

**Pull request closed without merging**